Website: Update Button Organization pattern (bug bash) (#2676)
hashicorp/design-system: website/docs/patterns/button-organization/partials/guidelines/alignment.md

![](/assets/patterns/button-organization/alignment-methods.png)
!!!

### Left alignment

In most scenarios, align buttons to the **left side** of a container or page. The primary button should be leftmost, followed by secondary, then tertiary.

- This reinforces natural reading patterns and a conversational "dialog-like" messaging strategy.
- When combined with a consistent ordering method, the primary button is prioritized in the reading direction.
- When considering different contexts, opting for a single method helps to create consistency between constrained and unconstrained components and patterns.

A common example of this alignment method is in [Modals](/components/modal) and page-level forms.

!!! Do

Align buttons to the left side of the container.

![Modal with buttons in the footer aligned to the left. The reading order goes from left to right, top to bottom.](/assets/patterns/button-organization/alignment-reading-pattern-comparison.png)

!!!

### Right alignment

> This alignment method is commonly used for page-level functions or actions that create and manage elements within a page. The primary button should be the farthest right, then secondary, then tertiary.

Reviewer: This now reads a little weird to me, WDYT about this?

```suggestion
In the case of page-level functions or actions that manage elements within the page, align buttons to the right of the page. The primary button should be the farthest right, then secondary, then tertiary.
```

Author: Way better, committing.
